pkg/controller/buildrun/buildrun_controller.go

```diff
@@ -190,6 +190,9 @@ func (r *ReconcileBuildRun) Reconcile(request reconcile.Request) (reconcile.Resu
 			buildRun.Status.BuildSpec = &build.Spec
 		}
 
+		buildRun.Status.LatestTaskRunRef = &lastTaskRun.Name
+		buildRun.Status.StartTime = lastTaskRun.Status.StartTime
+
 		if lastTaskRun.Status.CompletionTime != nil && buildRun.Status.CompletionTime == nil {
```

Comment:
Any reasons why you didnt checek in here if `StartTime` from the BuildRun is nil?

Reply:
In this logic, we used to update the info from taskrun to buildrun status.

There are several status, like:
- buildRun.Status.CompletionTime
- buildRun.Status.LatestTaskRunRef
- buildRun.Status. StartTime

In future, we may get more status from taskrun to build status.

We don't need to check nil for everyone, we just need to update by using them.

But sure, let us see if we need some nil check to make sure the robustness in future.

Thanks!